Maddori.Apple/Maddori.Apple/Global/Supports/SceneDelegate.swift

```diff
@@ -66,3 +66,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
 }
 
+extension SceneDelegate {
+    func logout() {
+        window?.rootViewController = LoginViewController()
+    }
```

Comment:
p1;

```suggestion
    func logout() {
        window?.rootViewController = UINavigationController(rootViewController: LoginViewController())
    }
```

로그아웃 후 재 로그인 할 때 Navigation이동이 안돼서 UINavigationController를 감싸줘야할 것 같아요 !

Reply:
[ec1908d](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/166/commits/ec1908d0e2d5a0f4b9ffc5b5da95fcb531694a72)

수정완료했습니다.